src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -964,6 +965,10 @@ else if (!Filter.CHECKBOX.equals(filter.type)) {
                     if (path != null && !f2) {//additional field
                         filter.setFilter2(new NameValue(path, ""));
                     }
+                    if (f1 && !f2 && Filter.DATE.equals(filter.type) && filter.isRange()) {
```

Comment:
When there's a range set and there's no end date--as was seen in at least one saved filter--let's just look waaaay in the future instead of requiring the user to set "and less than today"